Packs/Active_Directory_Query/Integrations/Active_Directory_Query/Active_Directory_Query.py

```diff
@@ -536,16 +533,25 @@ def free_search(default_base_dn, page_size):
     if attributes:
         attributes = ALL_ATTRIBUTES if attributes == 'ALL' else attributes.split(',')
 
+    page_cookie = args.get('page-cookie')
+    if args.get('page-size'):
+        page_size = arg_to_number(args['page-size'])
+        size_limit = page_size
```

Comment:
@yaakovi 
Do we prefer in this case to throw an exception for the user that he entered `page_size` and `size_limit`?